src/hotspot/cpu/aarch64/methodHandles_aarch64.cpp

```diff
@@ -185,7 +185,7 @@ address MethodHandles::generate_method_handle_interpreter_entry(MacroAssembler*
     return NULL;
   }
 
-  // r13: sender SP (must preserve; see prepare_to_jump_from_interpreted)
+  // r19_sender_sp: sender SP (must preserve; see prepare_to_jump_from_interpreted)
```

Comment:
There are two other references to r13 that either need removing or fixing
```
286   Register temp3 = r14;  // r13 is live by this point: it contains the sender SP
```
and
```
356     // Live registers at this point:
357     //  member_reg - MemberName that was the trailing argument
358     //  temp1_recv_klass - klass of stacked receiver, if needed
359     //  r13 - interpreter linkage (if interpreted)  ??? FIXME
360     //  r1 ... r0 - compiler arguments (if compiled)
```